### ISSUE TITLE
Breaks with multiline strings

### ISSUE THREAD

**Issue by kg-currenxie** [pascalre/vscode-yaml-sort#134]
```
---
sv:
  matter_mailer:
    created:
      subject: Nytt ärende skapat
      body: |
        Hej %{name},

        Tack för att du skapat ett ärende hos oss. Vi kommer att skicka ett paket till dig inom kort.

        Du kan följa ditt ärende på %{url}.

        Med vänliga hälsningar,
        %{app_name}
```

Error:
```
bad indentation of a mapping entry (4:37)

 1 | sv:
 2 |   matter_mailer:
 3 |     created:
 4 |       body: vscode-yaml-sort.block.0: Nytt ärende skapat
-----------------------------------------^
 5 |       title: Nytt ärende
```

Removing the body key works fine

**Comment by Wolfeg** [comment]
same problem, block processor not working.
turn it off in settings -> extensions -> yaml sort -> uncheck 'use block processor'

**Comment by pascalre** [comment]
Hi @kg-currenxie, this bug is fixed in 6.5.5. Please update and let me know if it works.
If you like this extension, please leave me a review on the [marketplace](https://marketplace.visualstudio.com/items?itemName=PascalReitermann93.vscode-yaml-sort&ssr=false#review-details).
